Demo/Demo/FeatureViewControllers/FeatureBaseViewController.swift

```diff
@@ -76,7 +86,7 @@ class FeatureBaseViewController: UIViewController {
     }
 
     func createOrder(completion: @escaping (String?) -> Void = { _ in }) {
```

Comment:
random nit: could this go in the BaseViewModel too?

Reply:
Done!